docs/reference/settings/security-hash-settings.asciidoc

```diff
@@ -0,0 +1,83 @@
+[float]
+[[hashing-settings]]
+==== User cache and password hash algorithms
+
+Certain realms store user credentials in memory. To limit exposure 
+to credential theft and mitigate credential compromise, cached user credentials 
```

Comment:
maybe `To limit exposure to credential theft and mitigate credential compromise, the cache only stored a hashed version of the user credentials`

Reply:
:+1: Thanks, I like that better too